crates/dock/tests/mutability_check/reassign_immutable/main.nail

```diff
@@ -0,0 +1,15 @@
+fn main() {
+    let a = 10;
+    a = 20;
```

Comment:
**Security Issue**: Attempt to reassign an immutable variable 'a', which can lead to runtime errors or unexpected behavior. <br> **Fix**: Modify the variable declaration to include 'mut' to allow reassignment. <br> **Code Suggestion**: 
 ```
 -    let a = 10;
 +    let mut a = 10;
 ```